CHANGELOG.md

```diff
@@ -5,63 +5,85 @@ CHANGELOG
+- Add `--suppress-validators` and `--validation-failure-level` parameters to `update` command.
 - Remove possibility to specify aliases for `ssh` command in the configuration file.
-- Distribute AWS Batch commands: `awsbhosts`, `awsbkill`, `awsbout`, `awsbqueues`, `awsbstat` and `awsbsub`
-  as a separate `aws-parallelcluster-awsbatch-cli` PyPI package.
+- Rename `createami` command to `build-image` command, remove `--ami-id`, `--os`, `--instance-type`,
+  `--ami-name-prefix`, `--custom-cookbook`, `--post-install`, `--no-public-ip`, `--cluster-template`, `--vpc-id`,
+  `--subnet-id`.
+- Add `--image-name`, `--config`, `--region` parameters to `build-image` command.
+- Add `delete-image` command with `--name`, `--region`, `--force` parameters.
+- Add `describe-image` command with `--name`, `--region` parameters.
+- Add `list-images` command with `--region`, `--color`parameters.
+- Add `export-cluster-logs`, `list-cluster-logs` and `get-cluster-log-events` commands to retrieve both CloudWatch Logs
+  and CloudFormation Stack Events. Add `export-image-logs`, `list-image-logs` and `get-image-log-events` commands to
+  retrieve both Image Builder Logs and CloudFormation Stack Events.
+- Distribute AWS Batch commands: `awsbhosts`, `awsbkill`, `awsbout`, `awsbqueues`, `awsbstat` and `awsbsub` as a
+  separate `aws-parallelcluster-awsbatch-cli` PyPI package.
+- Upgrade Python runtime used by Lambda functions in AWS Batch integration to python3.8.
+- Add timestamp suffix to CloudWatch Log Group name created for the cluster.
 - Remove `pcluster-config` CLI utility.
 - Remove `amis.txt` file.
-- Upgrade Python runtime used by Lambda functions in AWS Batch integration to python3.8.
-- Add timestamp suffix to CloudWatch Log Group name created for the cluster.  
 - Remove additional EBS volume attached to the head node by default.
 - Change NICE DCV session storage path to `/home/{UserName}`.
-- Use inclusive language for internal variable, parameters and CloudFormation Outputs:  
-  - Rename variable exported in the AWS Batch job environment from `MASTER_IP` to `PCLUSTER_HEAD_NODE_IP`.
-  - Rename all CloudFormation outputs from `Master*` to `HeadNode*`.
-  - Rename `NodeType` and tags from `Master` to `HeadNode`.
-- Remove `parallelcluster-` prefix from CloudFormation stack created by ParallelCluster.
+- Create a single ParallelCluster S3 bucket for each AWS region rather than for each cluster.
+- Adopt inclusive language
+  - Rename MasterServer to HeadNode in CLI outputs.
+  - Rename variable exported in the AWS Batch job environment from MASTER_IP to PCLUSTER_HEAD_NODE_IP.
+  - Rename all CFN outputs from Master* to HeadNode*.
+  - Rename NodeType and tags from Master to HeadNode.
+- Remove Ganglia support.
+- Add support for associating an existing Elastic IP to the head node.
+- Encrypt root EBS volumes and shared EBS volumes by default. Note that if the scheduler is AWS Batch, the root volumes
```

Comment:
Enhancement

Reply:
humm isn't this more like a change?

Reply:
I'd sell it as an enhanced security posture